src/pages/login.js

```diff
@@ -341,13 +57,25 @@ export const LoginFormContainer = () => {
                     </p>
                 </FormSubtitle>
             )}
+            {(emailtwoFAVerificationRequired || emailTwoFAIncorrect) && (
+                <FormSubtitle>
+                    <p>
+                        {i18n.t(
+                            'We have sent you an email with your authentication code, enter it below to log in',
```

Comment:
`We have sent you an email with your authentication code. Enter it below to log in.`